### DISCUSSION TITLE
codemagic static pages via yaml

### DISCUSSION THREAD

**Discussion by paul-charlton** [codemagic-ci-cd/codemagic-docs#1251] [Q&A] [answered]
Hi,
This [article ](https://blog.codemagic.io/build-and-host-your-flutter-web-apps-on-codemagic/) details how to setup a codemagic static page deployment using the UI.  And [here ](https://docs.codemagic.io/yaml-publishing/app-store-connect/) has the informaiton about using the yaml for various deployment options.  However, the codemagic static page is omitted from here.

Does anyone know the detail for configuring the yaml for a codemagic static deployment?

Thnaks in advance,
Paul.

**Comment by K-Codemagic** [comment, marked as answer]
Hi, we provide static pages for quick preview for educational purpose mostly. Advanced users should consider using professional solutions like Firebase Hosting to host sites.

So this is why we don't support static page deployment with codemagic.yaml file, but you can easily publish to Firebase Hosting using preinstalled Firebase CLI tools or to any other services.

**Reply by paul-charlton** [reply]
No worries - I had a feeling this was going to be the case.  We we're using the codemagic for a quick test case, proper hosting was going to need sorting out anyway!
